src/client/config_info.h

```diff
@@ -274,6 +274,7 @@ struct IOOption {
 struct CommonConfigOpt {
     bool mdsRegisterToMDS = false;
     bool turnOffHealthCheck = false;
+    uint32_t minimalOpenFiles = 65536;
```

Comment:
Put the description of the configuration item in a comment.

Reply:
fixed